requirements.txt

```diff
@@ -2,6 +2,7 @@ azureml-core>=1.15.0
 azureml-dataprep
 azureml-mlflow
 azureml-widgets
+azureml-tensorboard
```

Comment:
do we want this globally?

Reply:
it's needed to run the tensorboard notebook. is it fine if I just make a note to tell people to download it in the notebook itself?

Reply:
yes that's fine, other notebooks have "!pip install --upgrade packages" in them